src-vue/src/plugins/store.ts

```diff
@@ -200,6 +200,12 @@ async function _initializeApp(state: any) {
     // Enable dev mode directly if application is in debug mode
     if (await invoke("is_debug_mode")) {
         state.developer_mode = true;
+
```

Comment:
Actually, this is not the good place for this code; you should rather put it where dev mod is enabled/disabled.

Reply:
> Currently, when disabling dev mod, stripped style stays there.

Uh, that's the point :P

The striped menubar is not tied to "developer mode" (which shows/hides DevView) but whether the binary has been compiled in debug mode (`npm run tauri dev`) or release mode (`npm run tauri build`).

So the idea is that if you have two FlightCore windows open. One is the locally compiled build you're working on rn and the other one is an installed release build, they are both visually distinct from another.

Sorry for the confusion haha ^^